states.inc.php

```diff
@@ -271,6 +271,15 @@
     'possibleactions' => ['actCirclePlant', 'actRestart', 'actPassOptionalAction'],
   ],
 
+  ST_STIR_WATER_TANKS => [
+    'name' => 'stirWaterTanks',
+    'descriptionmyturn' => clienttranslate('${you} may choose to stir water tanks'),
```

Comment:
I would probably use the singular instead : "you may choose to stir the attached water tank"

Reply:
Done